### PR TITLE
Dashboard mobile/projects

### PR DESCRIPTION
**Pull request by heyitsmaimai**
need to merge w/ [#184](https://github.com/CoderDojo/cp-zen-frontend/pull/184)

### CODE REVIEW

File: src/dashboard/cd-dashboard-projects.vue [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <template>
   <div class="cd-dashboard-projects">
-    <h3 class="cd-dashboard-projects__header">Before your next event, here are some projects you can try</h3>
+    <h2 class="cd-dashboard-projects__header hidden-xs">Before your next event, here are some projects you can try</h2>
+    <h2 class="cd-dashboard-projects__header visible-xs">Here are some projects you can try</h2>
```
Comment (Wardormeur) [inline src/dashboard/cd-dashboard-projects.vue]: Hum, why this reduction of the sentence? To look like a one-liner ?

File: src/dashboard/cd-dashboard-projects.vue [hunk 2]
```diff
@@ -110,4 +112,22 @@
       margin: 32px 0;
     }
   }
+
+  @media (max-width: @screen-xs-max) {
+    .cd-dashboard-projects {
+      max-width: 100%;
+
+      &__divider {
+        border-color: #CAC4CE; ;
```
Comment (Wardormeur) [inline src/dashboard/cd-dashboard-projects.vue]: Same than before, random color

File: src/dashboard/cd-dashboard-projects.vue (hunk 1 above)
Comment (heyitsmaimai) [reply src/dashboard/cd-dashboard-projects.vue]: That's what was in de mobile template. Also bc it looks awkwardly long as a header in mobile imo

Comment (Wardormeur) [reply src/dashboard/cd-dashboard-projects.vue]: Then add trnaslations as well :)

**Pull request merged**